pkg/local_object_storage/blobstor/common/storage.go

```diff
@@ -23,3 +27,53 @@ type Storage interface {
 	Delete(DeletePrm) (DeleteRes, error)
 	Iterate(IteratePrm) (IterateRes, error)
 }
+
+// Copy copies all objects from source Storage into the destination one. If any
+// object cannot be stored, Copy immediately fails.
+func Copy(from, to Storage) error {
```

Comment:
Don't we have something similar in evacuation logic?

Reply:
evacuation overtakes data from one shard to another not dealing with `common.Storage`